### ISSUE TITLE
Per database persistence configuration

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ashish21590/redis#317]
```
I have a critical database that needs to sync to HD with appendonly log, but I 
really don't want this to happen for the rest of my databases on the same Redis 
instance (to save disk space and reduce latency).



```

Original issue reported on code.google.com by `don...@gmail.com` on 1 Sep 2010 at 9:08

**Comment by GoogleCodeExporter** [comment]
```
Redis has a very small memory footprint. This means you can very easily run 
multiple instances in parallel, where you configure them differently. This will 
also have a good impact on your latency requirement, because you will have a 
dedicated instance for your critical data.
```

Original comment by `pcnoordh...@gmail.com` on 1 Sep 2010 at 10:24
- Changed state: **WontFix**
